app/js/components/manageApps/ManageApps.jsx

```diff
@@ -155,14 +155,18 @@ export default class ManageApps extends React.Component {
       const apiBaseUrl = `/${applicationDistribution}/${url}/ws/rest`;
       this.requestUrl = '/v1/module/?v=full';
       axios.get(`${urlPrefix}/${apiBaseUrl}${this.requestUrl}`).then(response => {
-        response.data.results.forEach((data) => {
-          installedModules.push({
-            appDetails: data,
-            appType: 'module',
-            install: false
+        return Promise
+          .all(response.data.results.map(data => axios.get(`https://addons.openmrs.org/api/v1/addon/${data.packageName}`)
```

Comment:
Can the base url 'https://addons.openmrs.org/api/v1/addon' be externalized instead of being in source code directly? Could make future changes easy were it to change

Reply:
That would be awesome! 👍